scylladb/sphinx-scylladb-theme
# Extend hero box

Related issue https://github.com/scylladb/scylladb-docs-homepage/issues/11

Edits the hero box extension to make the call to action more visible.

## How to test

1. Build the docs.
2. Open http://127.0.0.1:5500/examples/hero-box/#hero-box-with-everything
3. Check how the new button will look like:

    ![image](https://github.com/scylladb/sphinx-scylladb-theme/assets/9107969/11420b10-4cf4-4eb7-a0d0-e6f2aa3aa646)
